test/runtests.jl

```diff
@@ -12,20 +12,26 @@ solvers = [:trunk, :lbfgs]
 
 for model in models
   for solver in solvers
-    stats = run_solver(solver, model, verbose=true)
+    stats = run_solver(solver, model, verbose=false)
     assert(all([stats...] .>= 0))
     reset!(model)
   end
 end
 
 # clean up the test directory
-here = dirname(@__FILE__)
-so_files = filter(x -> (ismatch(r".so$", x) || ismatch(r".dylib$", x)), readdir(here))
+@unix_only begin
+  here = dirname(@__FILE__)
+  so_files = filter(x -> (ismatch(r".so$", x) || ismatch(r".dylib$", x)), readdir(here))
 
-for so_file in so_files
-	rm(joinpath(here, so_file))
-end
+  for so_file in so_files
+    rm(joinpath(here, so_file))
+  end
 
-rm(joinpath(here, "AUTOMAT.d"))
-rm(joinpath(here, "OUTSDIF.d"))
+  rm(joinpath(here, "AUTOMAT.d"))
+  rm(joinpath(here, "OUTSDIF.d"))
+end
 
+# test benchmark helpers, skip constrained problems (hs7 has constraints)
+run_ampl_problem(:trunk, :dixmaanj, 0, verbose=false)
+probs = [:dixmaane, :dixmaanf, :dixmaang, :dixmaanh, :dixmaani, :dixmaanj, :hs7]
+bmark_and_profile(solvers, probs, 99, skipif=m -> m.meta.ncon > 0)
```

Comment:
No, but I wanted to increase coverage and didn't want the call to `performance_profiles()` to fail because there's no backend available. Apparently, such output is suppressed during tests.